docs/tendermint-core/validators.md

```diff
@@ -17,14 +17,73 @@ to post any collateral at all.
+
+- `pex:` boolean. This turns the peer exchange reactor on or off for a node. When `pex=false`, only the `persistent_peers` list is available for connection.
+- `persistent_peers:` a comma separated list of `nodeID@ip:port` values that define a list of peers that are expected to be online at all times. This is necessary at first startup because by setting `pex=false` the node will not be able to join the network.
+- `unconditional_peer_ids:` comma separated list of nodeID's. These nodes will be connected to no matter the limits of inbound and outbound peers. This is useful for when sentry nodes have full address books.
+- `private_peer_ids:` comma separated list of nodeID's. These nodes will not be gossiped to the network. This is an important field as you do not want your validator IP gossiped to the network.
+- `addr_book_strict:` boolean. By default nodes with a routable address will be considered for connection. If this setting is turned off (false), non-routable IP addresses, like addresses in a private network can be added to the address book.
+
+#### Validator Node Configuration
+
+| Config Option          | Setting                    |
+| ---------------------- | -------------------------- |
+| pex                    | false                      |
+| persistent_peers       | list of sentry nodes       |
+| private_peer_ids       | none                       |
+| unconditional_peer_ids | optionally sentry node IDs |
+| addr_book_strict       | false                      |
+
+The validator node should have `pex=false` so it does not gossip to the entire network. The persistent peers will be your sentry nodes. Private peers can be left empty as the validator is not trying to hide who it is communicating with. Setting unconditional peers is optional for a validator because they will not have a full address books.
+
+#### Sentry Node Configuration
+
+| Config Option          | Setting                                       |
+| ---------------------- | --------------------------------------------- |
+| pex                    | true                                          |
+| persistent_peers       | validator node, optionally other sentry nodes |
+| private_peer_ids       | validator node ID                             |
+| unconditional_peer_ids | validator node ID, optionally sentry node IDs |
+| addr_book_strict       | false                                         |
+
+The sentry nodes should be able to talk to the entire network hence why `pex=true`. The persistent peers of a sentry node will be the validator, and optionally other sentry nodes. The sentry nodes should make sure that they do not gossip the validator's ip, to do this you must put the validators nodeID as a private peer. The unconditional peer IDs will be the validator ID and optionally other sentry nodes.
+
+> Note: Do not forget to secure your node's firewalls when setting them up.
```

Comment:
Should we link other useful guides here?

https://kb.certus.one/
https://forum.cosmos.network/t/sentry-node-architecture-overview/454